src/app/users/getBulkUserActionsEmails.js

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable no-await-in-loop */
+/* eslint-disable no-restricted-syntax */
+const { searchForBulkUsersPage } = require('./utils');
+
+const getBulkUserActionsEmails = async (req, res) => {
+  const emails = req.session.emails;
+  if (!emails) {
+    return res.redirect('/users');
+  }
+  const users = [];
+
+  const emailsArray = emails.split(',');
+  for (const email of emailsArray) {
+    const result = await searchForBulkUsersPage(email.trim());
```

Comment:
If a user puts in an email containing a special character e.g. `+` which can appear in alias emails, `result.users` does not exist and an error is thrown. An object with `validationMessages` is returned, which the error within should be displayed rather than an error being thrown 👍

Tested separation with `,` as well as `, ` and both worked correctly 👍